README.md

```diff
@@ -36,7 +36,7 @@ cloud-provider specific code out of the Kubernetes codebase.
 | v1.28.2   | v1.28                  | -                      |
 | v1.29.2   | v1.29                  | -                      |
 | v1.30.1   | v1.30                  | -                      |
-| v1.31.0   | v1.31                  | -                      |
+| v1.31.1   | v1.31                  | -                      |
```

Comment:
I think we could have added v1.32.0 to the table as well since it is also released with v1.31.1.